spec/requests/users/manager_spec.rb

```diff
@@ -88,6 +88,9 @@
 
     context 'when themself' do
       before do
+        message_delivery = instance_double(GovukNotifyRails::Mailer::MessageDelivery)
```

Comment:
use the shared let mentioned [above](https://github.com/ministryofjustice/laa-court-data-ui/pull/58/files#r384459663)

Reply:
👍